pkg/crc/constants/constants.go

```diff
@@ -123,26 +122,18 @@ func EnsureBaseDirectoriesExist() error {
 	return os.MkdirAll(CrcBaseDir, 0750)
 }
 
-// IsBundleEmbedded returns true if the executable was compiled to contain the bundle
-func BundleEmbedded() bool {
-	executablePath, err := os.Executable()
-	if err != nil {
-		return false
-	}
-	extractor, err := binappend.MakeExtractor(executablePath)
-	if err != nil {
-		return false
-	}
-	return contains(extractor.AvalibleData(), GetDefaultBundle())
-}
-
-func contains(arr []string, str string) bool {
-	for _, a := range arr {
-		if a == str {
-			return true
+// BundlePresent returns true and path of the bundle
+// if bundle is available in current directory or
```

Comment:
It's not the current directory we want to look into, but the directory where the `crc` executable is.
In this scenario:
```
$ mkdir foo
$ cd foo
$ ../crc start
```
you want to look for the bundle in `..`, not in the current directory, which is `foo`. `https://golang.org/pkg/os/#Executable` should  be useful for this.